src/drawers/arcDrawer.ts

```diff
@@ -3,13 +3,19 @@
 module Plottable {
 export module Drawers {
   export class Arc extends Drawer {
+    private _isOutline: boolean;
 
-    constructor(dataset: Dataset) {
+    constructor(dataset: Dataset, isOutline = false) {
```

Comment:
It seems like it would be better to make a subclass for this instead of using a `boolean` parameter.